### DISCUSSION TITLE
How to Display the Filament FullCalendar Widget Inside a Resource Instead of the Dashboard

### DISCUSSION THREAD

**Discussion by robertox85** [saade/filament-fullcalendar#226] [Q&A]
Hello,

I would like to display the Filament FullCalendar widget inside a Resource instead of the dashboard. Could you please provide guidance or examples on how to achieve this?

**Comment by Daniel-Barragan-Torres** [comment]
From the little I Know, it´s not possible to show big Widgets in Resources. I spent a lotta time to figure this out. I had to create a separate page for it. Best of luck friend!
